etc/kayobe/kolla/globals.yml

```diff
@@ -61,3 +61,7 @@ prometheus_libvirt_exporter_tag: 7.0.1.4
 rabbitmq_tag: rocky
 storm_tag: rocky
 zookeeper_tag: rocky
+
+# Kayobe currently configures ntpd on each node in the overcloud
+enable_chrony: false
+enable_host_ntp: true
```

Comment:
I see they turned enabled_host_ntp to False in Rocky. What about the Kayobe ntp mechanism? Is the Kolla option as configurable?